tailwind.config.js

```diff
@@ -4,6 +4,8 @@ export const theme = {
   extend: {
     colors: {
       'primary-blue': '#1677FF',
+      'ant-icon-color': '#525F7F',
```

Comment:
Lets use a different name here without the ant part

```suggestion
      'ant-icon-color': '#525F7F',
```